controllers/license/license.js

```diff
@@ -21,14 +21,25 @@ exports.getBadgeFromData = (req, res) => {
         res: res
     };
 
+    let service = config.licence.mainService;
+    //ToDo: Implement multiple services and a fallback when there is no result
```

Comment:
same as in `executability.js`, this seems to be reappearing input validation code which should only be written once, if possible.